### PR TITLE
FIX: putObjectTagging

### PR DESCRIPTION
**Pull request by nicolas2bert**
- adding the right ACL permission
- renaming CORSHeaders variable

### CODE REVIEW

**Comment by nicolas2bert** [comment]
@ironman-machine ok

File: lib/api/apiUtils/authorization/aclChecks.js [hunk 1]
```diff
@@ -69,6 +69,12 @@ export function isObjAuthorized(bucket, objectMD, requestType, canonicalID) {
     if (objectMD['owner-id'] === canonicalID) {
         return true;
     }
+    // user is authorized if:
```
Comment (LaurenSpiegel) [inline lib/api/apiUtils/authorization/aclChecks.js]: I would say account instead of user

File: lib/api/apiUtils/authorization/aclChecks.js [hunk 2]
```diff
@@ -69,6 +69,12 @@ export function isObjAuthorized(bucket, objectMD, requestType, canonicalID) {
...
+    // - requesttype is "bucketOwnerAction" (example: for putObjectTagging) and
+    // - user is the bucket owner
```
Comment (LaurenSpiegel) [inline lib/api/apiUtils/authorization/aclChecks.js]: same

File: tests/functional/aws-node-sdk/test/object/putObjTagging.js [hunk 3]
```diff
@@ -126,5 +130,82 @@ describe('PUT object taggings', () => {
...
+                    objectNameAcl }, err => next(err)),
+                next => s3.putObjectTagging({ Bucket: bucketName,
+                      Key: objectNameAcl, Tagging: { TagSet: [
+                          {
+                              Key: 'key1',
+                              Value: 'value1',
+                          },
+                      ] },
+                    }, err => next(err)),
+            ], done);
```
Comment (bennettbuchanan) [inline tests/functional/aws-node-sdk/test/object/putObjTagging.js]: Shouldn't this test assert something? Or we just want to know that it completes without an error.

File: tests/functional/aws-node-sdk/test/object/putObjTagging.js [hunk 4]
```diff
@@ -126,5 +130,82 @@ describe('PUT object taggings', () => {
...
+
+        it('should put tag to an object in a bucket created with same ' +
+        'account', done => {
+            async.waterfall([
+                next => s3.putBucketAcl({ Bucket: bucketName, ACL:
+                  'public-read-write' }, err => next(err)),
+                next => otherAccountS3.putObject({ Bucket: bucketName, Key:
+                    objectNameAcl }, err => next(err)),
+                next => s3.putObjectTagging({ Bucket: bucketName,
+                      Key: objectNameAcl, Tagging: { TagSet: [
```
Comment (bennettbuchanan) [inline tests/functional/aws-node-sdk/test/object/putObjTagging.js]: The `Tagging` object is reused across these tests, perhaps you could define it as a variable.

Comment (nicolas2bert) [inline tests/functional/aws-node-sdk/test/object/putObjTagging.js]: Sure :) thanks.

File: tests/functional/aws-node-sdk/test/object/putObjTagging.js (hunk 3 above)
Comment (nicolas2bert) [inline tests/functional/aws-node-sdk/test/object/putObjTagging.js]: Yes exactly I just want it to no return any error :)

**Comment by nicolas2bert** [comment]
@ironman-machine ok

**Pull request merged**